app/controllers/v2/value_definitions_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module V2
+  # API for Value Definitions under Security Guides
+  class ValueDefinitionsController < ApplicationController
+    def index
+      render_json value_definitions
+    end
+    permission_for_action :index, Rbac::COMPLIANCE_VIEWER
+
+    def show
+      render_json value_definition
+    end
+    permission_for_action :show, Rbac::COMPLIANCE_VIEWER
+
+    private
+
+    def value_definitions
+      @value_definitions ||= authorize(resolve_collection)
+    end
+
+    def value_definition
+      # rubocop:disable Rails/FindById
+      @value_definition ||= authorize(resource.find_by!(id: permitted_params[:id]))
+      # rubocop:enable Rails/FindById
```

Comment:
As you're querying the table directly (not a view) you don't need to juggle around with `find_by(id: )`, you can use the `find` as it finds the primary key:
```suggestion
      @value_definition ||= authorize(resource.find(permitted_params[:id])
```